resources/decision_tree.json

```diff
@@ -12,12 +12,103 @@
         ]
       },
       "outcomeTrue": {
-        "nextNode": "gnomad"
+        "nextNode": "spliceAI"
       },
       "outcomeFalse": {
         "nextNode": "exit_lb"
       },
       "outcomeMissing": {
+        "nextNode": "spliceAI"
+      }
+    },
+    "spliceAI": {
+      "type": "BOOL_MULTI",
+      "fields": [
+        "INFO/CSQ/SpliceAI_pred_DS_AG",
```

Comment:
all integration test fail (SpliceAI plugin is not executed during integration tests):
```
Command output:
  classifying variants with decision tree ...
  Fieldtype is unavailable for field 'SpliceAI_pred_DS_AG' that is not present in the input vcf.
  java.lang.UnsupportedOperationException: Fieldtype is unavailable for field 'SpliceAI_pred_DS_AG' that is not present in the input vcf.
```